lib/surrealist/helper.rb

```diff
@@ -0,0 +1,15 @@
+module Surrealist
+  # A generic helper.
+  class Helper
+    class << self
+      # Determines if the class uses the Surrealist mixin.
+      #
+      # @param [Class] to be checked.
+      #
+      # @return [Boolean] if Surrealist is included in class.
+      def surrealist?(klass)
+        klass.included_modules.include?(Surrealist)
```

Comment:
It would be better to check it as `klass < Surrealist`

Reply:
Oh this is neat never knew this existed/worked